dev/Vagrantfile

```diff
@@ -0,0 +1,16 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.hostname = "vermont"
+  config.vm.box = "trusty_amd"
+  config.vm.box_url = "https://cloud-images.ubuntu.com/vagrant/trusty/current/trusty-server-cloudimg-amd64-vagrant-disk1.box"
+  config.vm.network "private_network", ip: "192.168.50.4"
```

Comment:
What is the purpose of this private network in this setup?
If the intent is that it is to provide an interface for vermont to be configured/tested against, it would be good make mention of this in the README, along with a example basic vermont.xml.

Reply:
Still to be discussed with @ogasser what a typical use case for vermont in a VM would be: Is it more preferable to capture test streams coming from the host or should the VM be in the public network so that it can be addressed from outside?

Reply:
I would see two typical use cases:
1. Capture raw traffic and export IPFIX
2. Collect IPFIX and store it in a database or send to ZeroMQ

The second scenario does not need any special network configuration as far as I can tell.

For the first scenario we need traffic which can be captured in the VM. I see three ways to do that:
- Generate traffic within the VM
- Provide pcap file to be replayed in the VM
- Configure the network such that the VM is able to capture the host's traffic

Comments? Suggestions?

Reply:
I would try to support what a typical user of vermont would initially try to achieve using the VM. Do you have any insights what's the typical use case?

Reply:
I think these are the two typical use cases which I mentioned above.

@nickbroon What do you think about the first scenario regarding traffic generation/forwarding?

Reply:
I think scenario 1 would probably be the most commonly used. That is have it listen to raw traffic on a public interface and then export ipfix flows.

Reply:
@mvonmaltitz Can you change the configuration to a public interface to account for scenario 1?

Reply:
@ogasser Configuration changed to public interface.